src/components/pills/c-pill/c-pill.ts

```diff
@@ -10,6 +10,14 @@ export class CPill {
     @bindable
     public color = 'var(--c_subOneMain)';
     @bindable
+    public textColor;
```

Comment:
Let's point these defaults to the matching css var so they aren't blank by default. This might very well have been messed up before you started work on this.
```
@bindable
public textColor = var(--pill-text-color);
```
after we should update the docs file above to say what the default value is.

Reply:
ok yea I'll add that